web/src/message_list_tooltips.js

```diff
@@ -234,4 +235,34 @@ export function initialize() {
             instance.destroy();
         },
     });
+
+    message_list_tooltip(".message_edit_notice", {
+        trigger: "mouseenter",
+        delay: LONG_HOVER_DELAY,
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "bottom",
+                    },
+                },
+            ],
+        },
+        onShow(instance) {
+            const $elem = $(instance.reference);
+            const edited_notice_str = $elem.attr("data-tippy-content");
+            const $markup = $("<span>").text(edited_notice_str);
+
+            if (page_params.realm_allow_edit_history) {
+                const second_line = $t({defaultMessage: "Click to view edit history."});
```

Comment:
Rather than `second_line`, maybe go with `hint_line_str` to parallel `edited_notice_str`, or something--it's just a good habit not to enumerate things in variables, so that future changes don't run into `third_line` or `fortieth_line` or something like that :)

Reply:
changed it